_squirtle/bash-rpc-prompt

```diff
@@ -1 +1 @@
-export PS1="\$(sq rpc -s) ${PS1}"
\ No newline at end of file
+export PS1="\$(sq rpc -p) ${PS1}"
```

Comment:
Is this firing for every invocation in the futurenet bash? I'm starting to think I may not like this. Need to play around with it a bit more.

Reply:
Also let's move this to the second PR (see below comments for splitting this PR into two PRs)